### PR TITLE
Refactor Shop / Room Shop Display

### PR DESCRIPTION
**Pull request by SerperiorBae**
- Again just Updating this to suit Wisps Colour Scheme and stop it from being so bland. >~<
- If you havent already seen it: Preview - http://prntscr.com/bm4r18
- Asked several users and they liked it.and received generally positive responses.

### CODE REVIEW

File: chat-plugins/economy.js [hunk 1]
```diff
@@ -10,6 +10,12 @@ const geoip = require('geoip-ultralight');
 geoip.startWatchingDataUpdate();
 
 let shopTitle = 'Wisp Shop';
+let shop-table = 'background-color: #ffc775 ; border: #c13a00 solid 1px ; color: #ff560e ; border-collapse: collapse ; padding: 5px ; width: 100%';
```
Comment (panpawn) [inline chat-plugins/economy.js]: const with uppercase ans underscores would look better aesthetically

Comment (SerperiorBae) [inline chat-plugins/economy.js]: You're actually not wrong tbh; was debating on doing this.

**Comment by AlainDuhHero** [comment]
aah pcool....
but if u do these additons to the custom.css as a custom css class
variables will get reduced n_n
less variables = more speed
Which is needed for realtime servers

**Comment by panpawn** [comment]
Ignore Lord Haji, he is banned from our repo for a reason

**Comment by CharizardtheFireMage** [comment]
@SerperiorBae Float submitted a PR on adding CSS to the shop so there is no use for this PR then.

**Comment by panpawn** [comment]
The shop has been restyled in https://github.com/jd4564/Pokemon-Showdown/pull/105

**Pull request closed without merging**